doc/compiling/README.md

```diff
@@ -43,6 +44,9 @@ General options and their default values:
 
 Library specific options:
 
+    SDL2_DLL                        - Only if building with SDL2 on Windows; path to libsdl2.dll
+    SDL2_INCLUDE_DIR                - Only if building with SDL2; directory where sdl.h is located
+    SDL2_LIBRARY                    - Only if building with SDL2; path to libsdl.a/libsdl.so/libsdl.lib
```

Comment:
it's `SDL2_INCLUDE_DIRS` / `SDL2_LIBRARIES` and the files are not called libsdl.a